pkg/cmd/roachtest/tests/latency_verifier.go

```diff
@@ -181,6 +181,10 @@ func (lv *latencyVerifier) pollLatencyUntilJobSucceeds(
 			lv.logger.Printf("unexpected status: %s, error: %s", status, info.GetError())
 			return errors.Errorf("unexpected status: %s", status)
 		}
+		if lv.targetSteadyLatency != 0 && lv.maxSeenSteadyLatency > lv.targetSteadyLatency {
```

Comment:
Note to self/other reviewers: maxSeenSteadyLatency is only set once we've already already established a latency below the targetSteadyLatency.